internal/cli/install.go

```diff
@@ -494,15 +498,20 @@ func installRunner(
 		AuthToken:     resp.Token,
 	}
 
+	// We set the ID to be "init" since it is the initial static runner
+	id := "init"
```

Comment:
This is an interesting idea. I wonder if it'd be more descriptive if we called this `static-runner` instead for its id?

Reply:
@demophoon what are your thoughts here? I personally like `static-runner` better, but I remember we were discussing this when syncing up on how we uniquely identify a runner for uninstalling. So if we do change it, just wanna remain in sync 😄

Reply:
I think that static-runner makes sense here. Probably better than using init since that term would then be overloaded in some of the workflows, namely kubernetes.

Reply:
@demophoon great! After your PR is merged up to this branch, I'll update it to use `static-runner`, and make sure the different platforms don't add an additional suffix `-runner`, so we don't end up with `waypoint-static-runner-runner` 😃

Reply:
TL;DR - I'll update this field to just `static`, to avoid weird naming of resources such as `waypoint-static-runner-runner`.

@briancain @demophoon I remember why this was tricky now. We originally wanted this to be named `waypoint-init-runner`, but the problem was that the Waypoint Helm chart applies `-runner` as a suffix to the names of all the resources created in Kubernetes. For example, with `id` set to `static-runner`, `kubectl get pods` returns the following after a clean install:

```terminal-output
% kubectl get pods
NAME                              READY   STATUS    RESTARTS      AGE
waypoint-server-0                 1/1     Running   0             23m
waypoint-static-runner-runner-0   1/1     Running   3 (22m ago)   22m
```

To align with this, I added `-runner` in the naming of resources for other platforms (the job name in Nomad, etc.).

Reply:
Cool, that makes a lot of sense @paladin-devops !